contributing/code/standards.rst

```diff
@@ -214,6 +214,8 @@ Naming Conventions
 * Use `snake_case`_ for configuration parameters and Twig template variables
   (e.g. ``framework.csrf_protection``, ``http_status_code``);
 
+* Use `snake_case`_ for configuration route names and paths;
```

Comment:
Instead of adding a new phrase, I think we should update the previous phrase as follows:

```rst
* Use `snake_case`_ for configuration parameters, route names and Twig template
  variables (e.g. ``framework.csrf_protection``, ``http_status_code``);
```

I wouldn't mention route paths because it depends (and for SEO can be important): `foo_bar` is considered a single word and `foo-bar` is considered as two different words. That's why it matters in URL paths.

Reply:
Got it, it's done. Thanks for the clarification!